includes/admin/upgrades/upgrade-functions.php

```diff
@@ -892,3 +897,31 @@ function give_v187_upgrades(){
 		}
 	}
 }
+
+/**
+ * Update Capabilities for Give_Worker User Role.
+ *
+ * This upgrade routine will update access rights for Give_Worker User Role.
+ *
+ * @since      1.8.8
+ */
+function give_v188_upgrades() {
```

Comment:
@mehul0810 We do not need an upgrade for capabilities.

<img width="449" alt="screen shot 2017-05-17 at 10 35 04 am" src="https://cloud.githubusercontent.com/assets/1784821/26139226/a29fb908-3aec-11e7-8c3e-152fdba76e8a.png">

WordPress will automatically find capabilities attached to `give_worker` role.
Can you test this without an upgrade?